.scalafmt.conf

```diff
@@ -1,4 +1,4 @@
-version = 3.7.8
+version = 3.7.17
```

Comment:
Was this needed?

Reply:
Seems like no, I'll revert.

Reply:
reverted

Reply:
I changed this to 3.7.10 because without that update, in sbt `++3.3.4 test:compile` fails with:

```
[error] (coreJS / Compile / scalafmt) org.scalafmt.sbt.ScalafmtSbtReporter$ScalafmtSbtError: scalafmt: /home/gregg/lucid/enumeratum/enumeratum-core/src/main/scala-3/enumeratum/EnumCompat.scala:14: error: [dialect scala3] pattern must be a value or have parens: EnumMacros.findValuesImpl[A]
[error]   protected inline def findValues: IndexedSeq[A] = ${ EnumMacros.findValuesImpl[A] }
[error]                                                                                    ^ [/home/gregg/lucid/enumeratum/enumeratum-core/src/main/scala-3/enumeratum/EnumCompat.scala]
```
there and a few other places.